core/src/main/res/values/strings.xml

```diff
@@ -418,9 +418,20 @@
     <string name="playback_pref_sum">Headphone controls, Skip intervals, Queue</string>
     <string name="network_pref">Network</string>
     <string name="network_pref_sum">Update interval, Download controls, Mobile data</string>
+
     <string name="pref_autoUpdateIntervallOrTime_title">Update Interval or Time of Day</string>
     <string name="pref_autoUpdateIntervallOrTime_sum">Specify an interval or a specific time of day to refresh the podcasts automatically</string>
     <string name="pref_autoUpdateIntervallOrTime_message">You can set an <i>interval</i> like \"every 2 hours\", set a specific <i>time of day</i> like \"7:00 AM\" or <i>disable</i> automatic updates altogether.\n\n<small>Please note: Update times are inexact. You may encounter a short delay.</small></string>
+
+    <string name="pref_autoUpdateIntervallOrTime_titleAct">Update podcast</string>
```

Comment:
The title here says `Intervall` with two `L`, which is German. I know that the existing strings above do that, too, but could you please switch to lower case with underscores and the typo fixed?